pylicenses/providers/piplocal.py

```diff
@@ -34,7 +35,8 @@ def get_package_metadata(self, pkg):
                 break
         return data
 
-    def find_license_files(self):
+    @staticmethod
```

Comment:
should not be a staticmethod (it is called on the class instance)

Reply:
I think this is a static method. What inside of this function is dependent on the class state?

Reply:
maybe not currently, it might in the future. It's definitely not a static method the way I implemented it, and I'd like to keep it that way. Thanks.

Reply:
removed